src/applications/mhv-landing-page/components/LandingPage.jsx

```diff
@@ -18,10 +21,11 @@ import NewsletterSignup from './NewsletterSignup';
 import WelcomeContainer from '../containers/WelcomeContainer';
 import { hasHealthData, personalizationEnabled } from '../selectors';
 
-const LandingPage = ({ data = {} }) => {
+const LandingPage = ({ data = {}, recordEvent = recordEventFn }) => {
   const { cards = [], hubs = [] } = data;
   const isUnverified = useSelector(isLOA1);
   const hasHealth = useSelector(hasHealthData);
+  const noHealthDataHeadline = 'You don’t have access to My HealtheVet';
```

Comment:
Can you just have this headline in NoHealthAlert and do the useEffect there too? It would be nice to keep that within the component that needs it. I just feel that headline belongs there. If the event recording needs to be here then consider having the headline in the alert and importing it here.

Reply:
I wanted to keep the `recordEvent` calls in the `LandingPage` component. This is the parent component of both `IdentityNotVerified` and `NoHealthAlert` components.  If the logic for calling recordEvent is moved to the children, I would need to update platform code (`IdentityNotVerified`)

Reply:
Having the headline at this level still does not feel right to me. I still suggest to just export it and use the export here.

Reply:
Done.